src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_badges.scss

```diff
@@ -0,0 +1,23 @@
+/*doc
+---
+title: badges
+name: badges
+category: Type
+---
+
+Badges for labeling things
+
+```html_example
+I am a normal user <span class="has-badge">Normal User</span>
+```
+
+*/
+
+.has-badge {
```

Comment:
The name suggests that this is a state, but it is a widget. I propose to change this to `badge`.